Doc/howto/unicode.rst

```diff
@@ -49,7 +49,7 @@ Some were true standards, defined by the International Organization for
 Standardization, and some were *de facto* conventions that were invented by one
 company or another and managed to catch on.
 
-255 characters aren't very many.  For example, you can't fit both the accented
+256 characters aren't very many.  For example, you can't fit both the accented
```

Comment:
I can't leave a comment on line 44, but I think we might want to also change it to say: 
```
hold 256 unique values ranging from 0 to 255.  ASCII codes only went up to 127, so some
```
I'm concerned that a beginner will see 256 for the first time and wonder where it comes from.

Reply:
I agree, but suggest adding 'one of' to get 'bytes could hold one of 256 unique values ...'.

Reply:
Agreed that that's an improvement.